README.md

```diff
@@ -169,18 +169,40 @@ milvus = vectorstore("milvus", host="localhost", port=19530)
 bot = bot("qna-over-docs", index="./examples/files/", vectorstore=milvus)
 ```
 
-## Exposing an API with FastAPI
+## Exposing an API with [langchain-serve](https://github.com/jina-ai/langchain-serve)
 
-You can also create a FastAPI app that will expose the bot as an API using the create_app function.
-Assuming you file is called `main.py` run `uvicorn main:app --reload` to run the API locally.
-You should then be able to visit `http://localhost:8000/docs` to see the API documentation.
+You can also expose the bot endpoints locally using langchain-serve. A sample file `api.py` is provided in the `megabots` folder.
 
-```python
-from megabots import bot, create_api
+To expose the API locally, you can do 
+```bash
+lc-serve deploy local megabots.api
+```
+
+You should then be able to visit `http://localhost:8000/docs` to see & interact with the API documentation.
 
-app = create_app(bot("qna-over-docs"))
+To deploy your API to the cloud, you can do
+```bash
+lc-serve deploy jcloud megabots.api
+```
+
+```text
+╭──────────────┬──────────────────────────────────────────────────────────────────────────────────────╮
+│ App ID       │                                 langchain-dec14439a6                                 │
+├──────────────┼──────────────────────────────────────────────────────────────────────────────────────┤
+│ Phase        │                                       Serving                                        │
+├──────────────┼──────────────────────────────────────────────────────────────────────────────────────┤
+│ Endpoint     │                      https://langchain-dec14439a6.wolf.jina.ai                       │
+├──────────────┼──────────────────────────────────────────────────────────────────────────────────────┤
+│ App logs     │                               dashboards.wolf.jina.ai                                │
+├──────────────┼──────────────────────────────────────────────────────────────────────────────────────┤
+│ Swagger UI   │                    https://langchain-dec14439a6.wolf.jina.ai/docs                    │
+├──────────────┼──────────────────────────────────────────────────────────────────────────────────────┤
+│ OpenAPI JSON │                https://langchain-dec14439a6.wolf.jina.ai/openapi.json                │
+╰──────────────┴──────────────────────────────────────────────────────────────────────────────────────╯
```

Comment:
Im not sure what exactly this is. Can you elaborate a bit? Was it meant to be a table?

Reply:
Yes, this is a markdown table. You can view the readme [here](https://github.com/deepankarm/megabots#exposing-an-api-with-langchain-serve).

Reply:
can you do it like the other one? Markdown has support for this. No need for text block

Reply:
I can hide it under a `Show Output` section that users can view only if needed. (Like we do [here](https://github.com/jina-ai/langchain-serve#enable-rest-apis))

Reply:
Done. Please check

Reply:
Not sure this is what I mean. Have a look at the other table at the read. But its still fine I can fix this afterwards so resolving this one 😁

Reply:
Yah, I definitely understand what you mean, but there 2 tables mean 2 different things. The table you have added is informative about the product and the table I've added just shows the copy-pasted output from CLI output.